README.md

```diff
@@ -135,6 +135,7 @@ We include [ ] and [x] to mark if the metirc is auto-installed in versa.
 | 36 | Speech enhancement-based CI-SDR [x] | se_snr | se_ci_sdr | [ESPnet](https://github.com/espnet/espnet.git) | |
 | 37 | Speech enhancement-based SAR [x] | se_snr | se_sar | [ESPnet](https://github.com/espnet/espnet.git) | |
 | 38 | Speech enhancement-based SDR [x] | se_snr | se_sdr | [ESPnet](https://github.com/espnet/espnet.git) | |
+| 39 | Audio Quality | pam | pam | [PAM](https://github.com/soham97/PAM/tree/main) | [Paper](https://arxiv.org/pdf/2402.00282)|
```

Comment:
We may consider using "Prompting Audio-Language Models (PAM)-based Audio Quality Evaluation" for the second column (metric name)